lib/sisimai/bite/json.rb

```diff
@@ -17,18 +17,18 @@ def index
         end
 
         # Convert from JSON object to Sisimai::Message
-        # @param         [Hash] mhead       Message header of a bounce email
-        # @param         [String] mbody     Message body of a bounce email(JSON)
-        # @return        [Hash, Undef]      Bounce data list and message/rfc822 part
-        #                                   or Undef if it failed to parse or the
-        #                                   arguments are missing
+        # @param         [Hash] mhead   Message header of a bounce email
+        # @param         [String] mbody Message body of a bounce email(JSON)
+        # @return        [Hash, Nil]    Bounce data list and message/rfc822 part
+        #                               or nil if it failed to parse or the
+        #                               arguments are missing
         def scan; return nil; end
 
         # @abstract      Adapt bounce object for Sisimai::Message format
-        # @param         [Hash] argvs       bounce object returned from each email cloud
-        # @return        [Hash, Undef]      Bounce data list and message/rfc822 part
-        #                                   or Undef if it failed to parse or the
-        #                                   arguments are missing
+        # @param         [Hash] argvs   bounce object returned from each email cloud
+        # @return        [Hash, Nil]    Bounce data list and message/rfc822 part
+        #                               or nil if it failed to parse or the
+        #                               arguments are missing
```

Comment:
__[rubocop]__
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->